src/app/shared/store/admin.actions.ts

```
@@ -566,3 +578,42 @@ export class ReinviteAreaAdminFail {
   constructor(public error: HttpErrorResponse) {}
 }
+
+export class GetModeratorProfile {
+  static readonly type = '[admin] Get Moderator Profile';
+
+  constructor() {}
+}
+
+export class ApproveWorkshopDraft {
+  static readonly type = '[admin] Approve Workshop Draft';
+  constructor(public draftId: string) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)